src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResourceTest.kt

```diff
@@ -25,29 +23,30 @@ class IncentiveReviewsResourceTest : SqsIntegrationTestBase() {
   @Autowired
   private lateinit var nextReviewDateRepository: NextReviewDateRepository
 
+  private val timeNow: LocalDateTime = LocalDateTime.now()
+
   @BeforeEach
   fun setUp(): Unit = runBlocking {
     offenderSearchMockServer.resetAll()
     prisonApiMockServer.resetAll()
 
     prisonerIepLevelRepository.deleteAll()
     nextReviewDateRepository.deleteAll()
-    persistPrisonerIepLevel(bookingId = 1234134, prisonerNumber = "A1234AA", iepTime = iepTime)
-    persistPrisonerIepLevel(bookingId = 1234135, prisonerNumber = "A1234AB", iepTime = iepTime)
-    persistPrisonerIepLevel(bookingId = 1234136, prisonerNumber = "A1234AC", iepTime = iepTime)
+    // Prisoners on Standard and not overdue
+    persistPrisonerIepLevel(bookingId = 1234134, prisonerNumber = "A1234AA", iepCode = "STD", iepTime = timeNow.minusDays(2))
+    persistPrisonerIepLevel(bookingId = 1234135, prisonerNumber = "A1234AB", iepCode = "STD", iepTime = timeNow.minusDays(1))
+    persistPrisonerIepLevel(bookingId = 1234136, prisonerNumber = "A1234AC", iepCode = "STD", iepTime = timeNow)
     // A prisoner on Basic
-    persistPrisonerIepLevel(bookingId = 1234137, prisonerNumber = "A1234AD", iepCode = "BAS")
+    persistPrisonerIepLevel(bookingId = 1234137, prisonerNumber = "A1234AD", iepCode = "BAS", iepTime = timeNow.minusDays(3))
     // A prisoner on Enhanced and overdue
-    persistPrisonerIepLevel(bookingId = 1234138, prisonerNumber = "A1234AE", iepCode = "ENH", iepTime = LocalDateTime.now().minusYears(2))
+    persistPrisonerIepLevel(bookingId = 1234138, prisonerNumber = "A1234AE", iepCode = "ENH", iepTime = timeNow.minusYears(2))
```

Comment:
last real review dates changed to all be non-identical so that sorting can be tested properly